example/kustomization.yaml

```diff
@@ -89,7 +89,7 @@ resources:
 - ../contrib/kserve/kserve
 - ../contrib/kserve/models-web-app/overlays/kubeflow
 
-#Pod Security Standarards enabled
+#Pod Security Standards
```

Comment:
Lets maybe link to the documentation in the comment https://kubernetes.io/docs/concepts/security/pod-security-standards/